src/bidiMapper/utils/Tests/stub.spec.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2021 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { assert, createStubInstance, SinonStubbedInstance } from 'sinon';
+
+export class Stub {
```

Comment:
Is a class with all static members being used to just so that everything is easier to import all at once? If so, I think it would be more idiomatic for consumers to use `import * as stub from './stub'`. I'd prefer not to define a class unless we really need one (i.e. it carries state and will be instantiated one or more times).

Reply:
Simplified and removed redundancy